src/main/webapp/app/shared/metis/posting-reactions-bar/posting-reactions-bar.directive.ts

```
@@ -121,6 +136,80 @@ export abstract class PostingsReactionsBarDirective<T extends Posting> implement
+                                        const response = await this.metisConversationService.createDirectConversation(userLogins[0]).toPromise();
+                                        newConversation = (response?.body ?? null) as Conversation;
+                                        if (newConversation) {
+                                            allSelections.push(newConversation);
+                                        }
+                                    } catch (error) {
+                                        return;
+                                    }
+                                } else {
+                                    try {
+                                        const response = await this.metisConversationService.createGroupConversation(userLogins).toPromise();
+                                        if (response && response.body) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)